app/controllers/sdg_management/relations_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
                .accessible_by(current_ability)
                .by_goal(params[:goal_code])
                .by_target(params[:target_code])
+               .by_revision_status(params[:revision_status])
```

Comment:
We usually use `has_filters` in controllers when dealing with this interface :thinking:. Are there complications preventing us from doing so here? Then we could add a couple of scopes to the `SDG::Relatable` concern.

Reply:
I did not realize we have the `HasFilter` option, never used it before, I think we could use here too. Thanks